packages/osd-optimizer/package.json

```diff
@@ -51,7 +51,7 @@
     "babel-loader": "^8.2.3",
     "css-loader": "^5.2.7",
     "file-loader": "^6.2.0",
-    "loader-utils": "^1.2.3",
+    "loader-utils": "^2.0.4",
```

Comment:
@AMoo-Miki should we revert the `package.json` bumps and add just the resolutions instead?

Reply:
I guess not for this PR to `main`

Reply:
It was manually backported to 1.x with #2995. It is better to have the updated `package.json` files.

Reply:
Probably something to keep track of for breaking changes for v3 do we have an issue for OpenSearch Dashboards that we wanted to keep track of stuff that are breaking changes?

Reply:
my method was adding the `v3.0.0` label